package.json

```diff
@@ -6,7 +6,7 @@
     "lint:js": "eslint .",
     "lint:js:fix": "yarn run lint:js --fix",
     "workflows:validate:tasklists": "ajv -s app/workflows/schemas/tasklist.schema.json -d app/workflows/lists/**/tasklist.yml --spec=draft2020",
-    "workflows:validate:sections": "ajv -s app/workflows/schemas/section.schema.json -d 'app/workflows/lists/**/sections/*.yml' --spec=draft2020",
+    "workflows:validate:sections": "ajv -s app/workflows/schemas/section.schema.json -d 'app/workflows/lists/**/sections/*.yml' --spec=draft2020 -r app/workflows/schemas/action.schema.json",
```

Comment:
Does `workflows:validate:sections` also validate actions now then? That seems odd?

Reply:
`workflows:validate:sections` has always validated actions, because we list the actions in our workflow in the `section.yml` files. What this does is tell the schema validator "validate the files at `app/workflows/lists/**/sections/*.yml` against `app/workflows/schemas/section.schema.json`, and the referenced schema you need is at `app/workflows/schemas/action.schema.json`".

We could just as easily nest all our definitions in one file, but it's likely to get unwieldy and hard to grok what it's specifying.